api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -74,25 +69,56 @@ public void runBeforeEachTest() {
 			adminService.setImplementationIdHttpClient(implementationHttpClient);
 			cacheManager = Context.getRegisteredComponent("apiCacheManager", CacheManager.class);
 		}
-		
+
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldReturnSystemInformation() {
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(5, systemInfoMap.size());
```

Comment:
is there something more specific we can assert on in addition? what is it that you expect to be in this systemInformation map?

Reply:
Hey @teleivo, Coveralls indicate that the lines are covered. Can you please elaborate why more assertions are needed?
Thank you :)

Reply:
Do we need to assert each member which the map contains?

Reply:
you only assert that the map contains 5 elements, you do not know whats inside. your test might pass by accident. maybe now it is not an accident and really does contain the elements you expect but if we refactor the AdminService in the future we might break our promises without noticing it due to an assertion that is not specific enough.

Reply:
Ok. :) 
Thank you!